brianlong/www.validators.app
# 183408113 active stake front component

#### What's this PR do?
- create cluster stat model with active stake, and display it on the front component

#### How should this be manually tested?
- run migrations
- run tests
- run daemons and sidekiq to check if new CLusterStat

#### What are the relevant tickets?
-[ URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183408113)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
